docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/architecture.md

```diff
@@ -0,0 +1,148 @@
+
+Traffic routing in the Amazon EKS VPC requires the following mapping for hybrid nodes:
+
+- Route table entries mapping hybrid node CIDR ranges to VPN endpoint.  
+  For example, Hybrid Node CIDR 10.200.0.0/16 → VPN endpoint 172.16.0.1.
+
+- Route table entries mapping hybrid pod CIDR ranges to VPN endpoint.  
+  For example, Hybrid Pod CIDR 192.168.0.0/16 → VPN endpoint 172.16.0.1.
+
+- For AWS Direct Connect, map traffic to appropriate private subnet CIDR.  
+  For example, both CIDRs 10.200.0.0/16 & 192.168.0.0/16 → Private subnet 172.16.1.0/24.
+
+For AWS VPNs, configure two static routes for each of the following CIDRs:
+
+- Hybrid Node CIDR block.  
+  For example, Hybrid Node CIDR 10.200.0.0/16 → VPN endpoint 172.16.0.1.
+
+- Hybrid Pod CIDR block.  
+  For example, Hybrid Pod CIDR 192.168.0.0/16 → VPN endpoint 172.16.0.1.
+
+If you're using a Virtual Private Gateway or Transit Gateway, route propagation can be enabled to automatically populate
+your VPC route tables. Ensure you verify your route tables after propagation.
+
+#### On-Premises and Edge Locations
+
+For on-premises and edge VPNs, set up IPsec Phase 1 tunnels with Phase 2 security associations for the following:
+
+- Hybrid Node subnet to EKS VPC CIDR.  
+  For example, Hybrid Node Subnet 10.201.0.0/16 → EKS VPC CIDR 10.100.0.0/16.
+
+- Hybrid Node pod CIDR to EKS VPC CIDR.  
+  For example, Hybrid Node Pod CIDR 192.168.0.0/16 → EKS VPC CIDR 10.100.0.0/16.
+
+You should also configure Border Gateway Protocol (BGP) or static routes on your on-premises or edge location router to
+ensure network traffic reaches the correct hybrid nodes. For static routing, this is explained in more detail during the
+[Configure Hybrid Node Networking for VPN Solutions](./create-hybrid-node-pools.md#configure-hybrid-node-networking-for-vpn-solutions)
+steps.
+
+A route must exist to send all traffic destined for the Amazon EKS VPC through a centralized VPN gateway, or
+alternatively, a unique VPN server IP can be defined for each hybrid node during the
+[Create Hybrid Node Pool](./create-hybrid-node-pools.md#create-hybrid-node-pool) steps.
+
+## Operating System Compatibility
+
+Palette supports the same operating systems as AWS. Refer to
+[Prepare operating system for hybrid nodes](https://docs.aws.amazon.com/eks/latest/userguide/hybrid-nodes-os.html) for
+details.
+
+## Supported Edge Hosts
+
+Palette allows you to use your edge hosts as your Amazon EKS Hybrid Nodes. Your edge hosts need to be registered with
+Palette before you can add them to your node pools.
+
+If you want to use your edge hosts as Amazon EKS Hybrid Nodes, they must have been registered using one of the following
+methods:
+
+- [Agent Mode](../../../../deployment-modes/agent-mode/agent-mode.md)
+- [Provider images](../../../edge/edgeforge-workflow/palette-canvos/build-provider-images.md)
+  - Provider images are [Kairos-based images](https://kairos.io/) containing the OS and the desired Kubernetes versions.
+
```

Comment:
@benradstone please add a warning indicating that the edge host's package manager must have an up-to-date package index, e.g. by running `sudo apt-get update` for Ubuntu and equivalent commands for other supported OSs. This is required for nodeadm install to properly download certain packages like containerd, the EKSD kubelet, and others.

Reply:
Will add this in 👍